hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/config/NodeConfig.java

```diff
@@ -101,4 +103,6 @@ public record NodeConfig(
         @ConfigProperty(HEDERA_PREFETCH_CODE_CACHE_TTL_SECS) int prefetchCodeCacheTtlSecs,
         @ConfigProperty(STATS_CONS_THROTTLES_TO_SAMPLE) List<String> consThrottlesToSample,
         @ConfigProperty(STATS_HAPI_THROTTLES_TO_SAMPLE) List<String> hapiThrottlesToSample,
-        @ConfigProperty(HEDERA_RECORD_STREAM_SIDE_CAR_DIR) String sidecarDir) {}
+        @ConfigProperty(HEDERA_RECORD_STREAM_SIDE_CAR_DIR) String sidecarDir,
+        @ConfigProperty(GRPC_WORKFLOWS_PORT) int workflowsPort,
+        @ConfigProperty(GRPC_WORKFLOWS_TLS_PORT) int workflowsTlsPort) {}
```

Comment:
Do `GRPC_WORKFLOWS_PORT` and `GRPC_WORKFLOWS_TLS_PORT` ports map to the new workflows, or are these the mono-service workflows? I guess I'm suggesting that the name be more explicit, e.g. `GRPC_HEDERA_APP_PORT`. That said, we've unofficially decided on `workflows` as the word to indicate 'this is the new app workflow', so this is probably fine

Reply:
yeah , named because everyone is aware Workflows are in new code